Dockerfile

```diff
@@ -17,9 +17,10 @@ FROM $base_image
 ENV GOVUK_APP_NAME=asset-manager
 
 # TODO: move ClamAV into a completely separate service.
-RUN install_packages clamav clamav-daemon clamdscan shared-mime-info && \
-    rm -fr /etc/clamav/* && \
-    mkdir -p /var/run/clamav && \
+RUN install_packages wget shared-mime-info && \
+    wget https://www.clamav.net/downloads/production/clamav-1.2.1.linux.x86_64.deb && \
```

Comment:
Looks like you're hardcoding amd64 here — this means developers will no longer be able to build natively on arm64 laptops (which they pretty much all have these days). Can you make it work for both architectures?

If not, that's ok for now (not ideal but ok) but you'd need to just check this cross-builds an amd64 image ok from an M1 Mac. (If that's already how you're working then you've already got it covered!)

Reply:
Just had a quick look myself and disappointingly Cisco doesn't seem to release arm64 packages itself :/ Their Docker images sometimes (but weirdly inconsistently) have arm64 e.g. [1.2.0](https://hub.docker.com/layers/clamav/clamav-debian/1.2.0-6/images/sha256-54a7e4404aff642d6fd611f8f2f73bf6c198e1dfcc0e533d29c226622f50407d?context=explore). That might be usable (`COPY --from`) but it's a different approach. Probably don't worry about it for now I guess.

Reply:
Good point here - i could probably download the source https://www.clamav.net/downloads/production/clamav-1.2.1.tar.gz - but building it make introduce complexity 
but i feel they will be able to build the dockerfile on M processors as i was able to do so.

Reply:
Oof yeah agree not worth trying to build it ourselves, that'd be a real extra maintenance burden that we could do without.